config/routes.rb

```diff
@@ -2620,23 +2620,25 @@
   # Enablement for the REST API
 
   # Semantic Versioning Regex for API, i.e. vMajor.minor.patch[-pre]
-  API_VERSION_REGEX = /v[\d]+(\.[\da-zA-Z]+)*(\-[\da-zA-Z]+)?/
+  API_VERSION_REGEX = /v[\d]+(\.[\da-zA-Z]+)*(\-[\da-zA-Z]+)?/ unless defined?(API_VERSION_REGEX)
 
   # OPTIONS requests for REST API pre-flight checks
   match '/api/*path' => 'api#handle_options_request', :via => [:options]
 
   get '/api(/:version)' => 'api#show_entrypoint', :format => 'json', :version => API_VERSION_REGEX
 
-  API_ACTIONS = {
-    :get    => "show",
-    :post   => "update",
-    :put    => "update",
-    :patch  => "update",
-    :delete => "destroy"
-  }.freeze
+  unless defined?(API_ACTIONS)
+    API_ACTIONS = {
+      :get    => "api#show",
+      :post   => "api#update",
+      :put    => "api#update",
+      :patch  => "api#update",
+      :delete => "api#destroy"
+    }.freeze
+  end
 
   def action_for(verb)
-    "api##{API_ACTIONS[verb]}"
+    API_ACTIONS[verb]
```

Comment:
@kbrock this was kept deliberately general because it will soon be `"possibly_something_else#show"`, as in https://github.com/ManageIQ/manageiq/pull/9868/files#diff-21497849d8f00507c9c8dcaf6288b136R2597